### PR TITLE
Add upgrade troubleshooting section

### PR DESCRIPTION
**Pull request by mterhar**
This is based on my recent experience.

### CODE REVIEW

**Comment by khorne3** [comment]
@mterhar Made some small edits FYI

**Review by khorne3 — APPROVED** [review]

**Review by bpmct — APPROVED** [review]
Thanks @mterhar. This ended up helping me today. 

Do you think it would make sense to mention that you may need to re-point your DNS records to the new ingress IP if you don't have a static IP?

**Comment by khorne3** [comment]
@mterhar are you ok w/ me merging this?

File: setup/updating.md [hunk 1]
```diff
@@ -101,3 +101,6 @@ If this happens, we recommend uninstalling and reinstalling:
     --wait --install --force --version 1.16.0 \
     coder coder/coder -f current-values.yml
     ```
+
+    The ingress may attach to a new public IP address so you may have to update 
+    the host and devurl IP addresses in your DNS provider.
```
Comment (bpmct) [inline setup/updating.md:106]: 🙏🏼

**Review by bpmct — APPROVED** [review]

**Pull request merged**